tools/crowdin/actions/pr-creator/index.js

```diff
@@ -0,0 +1,85 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+(async () => {
+  try {
+    const branch = core.getInput('branch-name');
```

Comment:
```suggestion
    const token = core.getInput('github-token');
    if (!token) {
      core.setFailed('Must provide an API token');
    }
    const branch = core.getInput('branch');
```

- Need to get the token out of the input to pass to a later suggestion.
- `action.yml` and the actual workflow files use `branch` instead of `branch-name`.